src/styles/blocks/card.scss

```
@@ -0,0 +1,126 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: $main-card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: $main-background-color;
+  margin: 1px;
+
+  @include transition;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    height: 36px;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    color: $main-text-color;
+    margin: 40px 16px 4px;
+
+    @include transition;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    height: 14px;
+    line-height: 14px;
+    font-size: 10px;
+    color: $small-text-color;
+    margin: 0 84px 16px 16px;
+  }
+}
+
+.price {
```

and move as nesting for `card` selector (similar you did with `&__product-code`
```suggestion
.card__price {
```